jujubundlelib/references.py

```diff
@@ -14,7 +14,7 @@
 # The following regular expressions are the same used in juju-core: see
 # http://bazaar.launchpad.net/~go-bot/juju-core/trunk/view/head:/charm/url.go.
```

Comment:
Could you please update this URL, from which these regexps are taken? We should really use the updated one on github.